storage/client/file/src/main/java/com/azure/storage/file/FileStorageClient.java

```diff
@@ -17,53 +17,53 @@
 import java.util.Map;
 
 /**
- * This class provides a fileServiceAsyncClient that contains all the operations for interacting with a file account in Azure Storage.
- * Operations allowed by the fileServiceAsyncClient are creating, listing, and deleting shares and retrieving and updating properties
+ * This class provides a fileStorageAsyncClient that contains all the operations for interacting with a file account in Azure Storage.
```

Comment:
Change this sentence to "This class provides operations for interacting with a file account in Azure Storage.

Reply:
Done.